### DISCUSSION TITLE
Welcome to AdGuardHome Discussions!

### DISCUSSION THREAD

**Discussion by ameshkov** [AdguardTeam/AdGuardHome#2726] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by MomenMamdouh** [comment]
Please can you add a time-based filtering option to block websites on specific days or hours?

**Reply by ameshkov** [reply]
Yeah, we have such a feature request: #951

**Comment by pitta111** [comment]
May I ask how can I make Adguardhome to an ipk file? I hope I can install it and run it on Openwrt. Thanks.
